src/OpenTelemetry.Instrumentation.Hangfire/.publicApi/netstandard2.0/PublicAPI.Unshipped.txt

```diff
@@ -1,9 +1,9 @@
 OpenTelemetry.Trace.HangfireInstrumentationOptions
-OpenTelemetry.Trace.HangfireInstrumentationOptions.DisplayNameFunc.get -> System.Func<Hangfire.BackgroundJob, string>
+OpenTelemetry.Trace.HangfireInstrumentationOptions.DisplayNameFunc.get -> System.Func<Hangfire.BackgroundJob!, string!>!
```

Comment:
Simplify this line: https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/b3c7ae771f77a8661db996a3e2b8e91e3242b281/src/OpenTelemetry.Instrumentation.Hangfire/Implementation/HangfireInstrumentationJobFilterAttribute.cs#L64-L65

to this since `DisplayNameFunc` is non-nullable now?
```csharp
Func<BackgroundJob, string> displayNameFunc = this.options.DisplayNameFunc;
```

Reply:
Fixed 462c6fdd6ba42efa280218085e7bdda737d70616